test/test.js

```diff
@@ -118,11 +126,10 @@ async function fileTest(t, sourceID, resultID, pluginOptions, ...plugins) { // e
 	};
 	const {code} = await transformFileAsync(sourceFile, {...babelrc, plugins});
 	const {code: result} = await transformFileAsync(resultFile, babelrc);
-
 	t.is(code, result);
 }
 
-test.serial('errors', t => {
+test('errors', t => {
```

Comment:
```suggestion
test.serial('errors', t => {
```

`test.serial` here ensures no other tests run while this one is running.  This is done because the test patches `console.error`, we want to ensure other tests run with the real `console.error`.

Reply:
Sorry I didn't mean to remove that